player/lua/osc.lua

```diff
@@ -1939,13 +1720,13 @@ local function osc_init()
         function ()
             mp.commandv("playlist-prev", "weak")
             if user_opts.playlist_osd then
-                show_message(get_playlist(), 3)
+                mp.command("show-text ${playlist} 3000")
             end
         end
     ne.eventresponder["shift+mbtn_left_up"] =
-        function () show_message(get_playlist(), 3) end
+        function () mp.command("show-text ${playlist} 3000") end
     ne.eventresponder["mbtn_right_up"] =
-        function () show_message(get_playlist(), 3) end
+        function () mp.command("show-text ${playlist} 3000") end
```

Comment:
Could we assign one function?

``` lua
ne.eventresponder["shift+mbtn_left_up"] = ne.eventresponder["mbtn_right_up"] = ...
```

Reply:
That's not valid Lua. Also we're going to replace them different script-opts afterwards anyway.